js/quote_tool.js

```diff
@@ -1 +1,12 @@
-// Qouting tool logic should be placed here.
\ No newline at end of file
+//  this is a validation of labels; if in the inputs there are one or more characters,
```

Comment:
Please start with capita T, and mage sure spaces are the same on both lines.

also multi line commenst should go 
```
/**
 * First line
 * Second line
 */
```